auto.def

```diff
@@ -110,6 +110,8 @@ options {
   with-zstd:path            => "Location of Zstandard"
 # System
   with-sysroot:path         => "Target system root"
+# Misc
+  cflags-record=1           => "Record the cflags used to build neomutt"
```

Comment:
Please use a negative, since the default is true: “Disable recording compiler flags in the output of neomutt -v” or something

Reply:
:+1:.